src/components/overlay_mask/overlay_mask.tsx

```diff
@@ -63,9 +63,19 @@ export const EuiOverlayMask: FunctionComponent<EuiOverlayMaskProps> = ({
     };
   }, []);
 
+  useEffect(() => {
+    if (typeof document !== undefined) {
```

Comment:
```suggestion
    if (typeof document !== 'undefined') {
```

`typeof` always returns a string